app/models/comment.rb

```diff
@@ -7,6 +7,8 @@ class Comment < ApplicationRecord
   validates :text, presence: true
 
   def update_comments
-    update(comments_counter: comments.count)
+    post = self.post
+    post.update(comments_counter: post.comments.count)
+    comments.save
```

Comment:
- [ ] The same happens here, it is safe to remove the `post = self.post`. However, there's an exception raised due to the last line `comments.save`, this is because your `Comment` model doesn't have a property called `comments`:

![image](https://github.com/AbbasSarwar/BlogApp/assets/77069011/cd1784ea-e275-4cf7-a09b-37cc6d0cbbc5)

Please consider removing this since the `update` method already persists in the update to the database! 🤓 🔧